stable_baselines3/common/logger.py

```diff
@@ -230,43 +266,49 @@ def make_output_format(_format: str, log_dir: str, log_suffix: str = '') -> KVWr
 # API
 # ================================================================
 
-def logkv(key: Any, val: Any) -> None:
+def logkv(key: Any, val: Any, exclude: Union[str, Tuple[str]] = None) -> None:
     """
     Log a value of some diagnostic
     Call this once for each diagnostic quantity, each iteration
     If called many times, last value will be used.
 
     :param key: (Any) save to log this key
     :param val: (Any) save to log this value
+    :param exclude: (str or tuple) outputs to be excluded
     """
-    Logger.CURRENT.logkv(key, val)
+    if isinstance(exclude, str):
+        exclude = exclude,
+    Logger.CURRENT.logkv(key, (val, exclude))
```

Comment:
not sure to get why you pass a tuple and not an extra argument...

Reply:
Well this way we have support for multiple exclusions

Reply:
I meant:
```python
logkv(key, val, ['one', 'two', 'three'])
```
would work, no?
my remark is more about `(val, exclude)` instead of `val, exclude`

Reply:
because the exclusion would still be kept in the dictionary and easier to process

Reply:
Would you like a `self.exclusions` along `self.name2val` instead?

Reply:
yes, please